Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.py

```diff
@@ -172,11 +172,13 @@ def list_secrets_command():
     })
 
 
-def list_secrets(engine_path, version):
+def list_secrets(engine_path, version, folder=None):
     path = engine_path
 
     if version == '2':
         path += '/metadata'
+        if folder:
+            path += '/' + folder
```

Comment:
Make sure `folder` does not already contain a `/` or use `path.join`

Reply:
good catch - added